applications/openshift/scc/scc_limit_network_namespace/rule.yml

```diff
@@ -26,7 +26,7 @@ references:
     nerc-cip: CIP-003-8 R6,CIP-004-6 R3,CIP-007-3 R6.1
     nist: CM-6,CM-6(1)
     pcidss: Req-2.2
-    srg: SRG-APP-000516-CTR-001325
+    srg: SRG-APP-000142-CTR-000330
```

Comment:
Just missed the reference initially?

Reply:
`SRG-APP-000516-CTR-001325` is a catch-all control that says "Container platform components must be configured in accordance with the security configuration settings based on DoD security configuration or implementation guidance, including SRGs, STIGs, NSA configuration guides, CTOs, and DTMs.". The one used now is SRG-APP-000142-CTR-000330 which says Container images instantiated by the container platform must execute using least privileges. and I think that's a much better fit.

Reply:
Ok - that makes sense.